scss/modules/_filters.scss

```diff
@@ -145,6 +155,39 @@
     height: auto;
   }
 
+  .combo--filter--mini {
```

Comment:
Not sure: does this make more sense here or in `_form-styles.scss` with the other `.combo` styles? I _think_ there, but not sure.